NOM_visualizations/README.md

```diff
@@ -0,0 +1,7 @@
+# Exploring NOM Metadata and Visualization via the NMDC Runtime API
+
+This folder includes two notebooks (in R and Python) that demonstrate how metadata from natural organic matter (NOM) can be gathered via the [NMDC-runtime API](https://api.microbiomedata.org/docs) and analyzed.
+
+## Python
+- [Static rendered Jupyter notebook](https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/main/NOM_visualizations/python/nom_data.ipynb). _Viewing only, not editable_
+- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/microbiomedata/nmdc_notebooks/blob/main/NOM_visualizations/python/nom_data.ipynb).This is the recommended way to interact with the notebook. _You need a google account to use this option_
```

Comment:
You might want to add a note that although google colab does work, it will be slow.  And point people to the other option.  You can see the taxonomy readme for that language.  You can test out the google colab here: https://colab.research.google.com/github/microbiomedata/nmdc_notebooks/blob/33-create-python-notebook-to-explore-processed-data-for-a-second-omics-data-type/NOM_visualizations/python/nom_data.ipynb#scrollTo=TN34Q-BwMAg1